core-java-modules/core-java-dates-operations/src/test/java/com/baeldung/date/comparison/DateComparisonUtilsUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.date.comparison;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+public class DateComparisonUtilsUnitTest {
+
+	private Date toDate(LocalDateTime localDateTime) {
+		return Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant());
+	}
+
+	Date day1Morning = toDate(LocalDateTime.of(2019, 10, 19, 6, 30, 40));
+	Date day1Evening = toDate(LocalDateTime.of(2019, 10, 19, 18, 30, 50));
+	Date day2Morning = toDate(LocalDateTime.of(2019, 10, 20, 6, 30, 50));
+
+	@Test
+	void givenDatesWithDifferentTime_whenIsSameDay_thenReturnsTrue() {
+

```

Comment:
Again formatting is off and can we remove the first blank lines in the test methods?